fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/HeaderNestingJsonBuilderStrategy.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.client.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import org.asynchttpclient.Response;
+
+import java.util.function.Function;
+
+/**
+ * A strategy that takes a JSON Node built from a Response, nests it and makes it a peer with the Response headers.
+ */
+public class HeaderNestingJsonBuilderStrategy implements Function<Response, JsonNode> {
```

Comment:
@garyluoex, No problem. @michael-mclawhorn Do I have the permission to remove it?